### PR TITLE
Use CPUID to identify processors instead of PCI Dev ID 

### PR DESCRIPTION
**Pull request by jun2zhou**
Add cpuid printing

Signed-off-by: Zhou Jun,  jun2.zhou@intel.com

### CODE REVIEW

**Comment by npmitche** [comment]
Do we want to store the CPUID like we are doing with the VID/DID/RID so we don't have to call it again if we need it for a module or HAL?

File: chipsec_main.py [hunk 1]
```diff
@@ -461,10 +461,12 @@ def log_properties(self):
         logger().log("[CHIPSEC] Python  : {} ({})".format(platform.python_version(),
                                                           "64-bit" if self.PYTHON_64_BITS else "32-bit"))
         logger().log("[CHIPSEC] Helper  : {} ({})".format(*self._cs.helper.helper.get_info()))
-        chip_info = "[CHIPSEC] {:8}: {}\n[CHIPSEC]      VID: {:04X}\n" \
+        chip_info = "[CHIPSEC] {:8}: {}\n[CHIPSEC]      CPUID: {}\n[CHIPSEC]      VID: {:04X}\n" \
```
Comment (frinzell) [inline chipsec_main.py:464]: Align to colon :

Comment (jun2zhou) [reply chipsec_main.py]: resolved

**Comment by jun2zhou** [comment]
changed to ":" aligned display,

cmds tested:

python3 chipsec_main.py

python3 chipsec_main.py -m misc.cpuidtest

python3 chipsec_util.py cpu info

**Comment by jun2zhou** [comment]
> Do we want to store the CPUID like we are doing with the VID/DID/RID so we don't have to call it again if we need it for a module or HAL?

let me rethink about that.

**Comment by frinzell** [comment]
> > Do we want to store the CPUID like we are doing with the VID/DID/RID so we don't have to call it again if we need it for a module or HAL?
> 
> let me rethink about that.

No, We can address this at a later date.

**Review by npmitche — APPROVED** [review]

**Review by dscott90 — APPROVED** [review]

**Pull request merged**